pysatNASA/instruments/icon_fuv.py

```diff
@@ -148,7 +132,43 @@ def clean(self):
                                       supported_tags=download_tags)
 
 
-def load(fnames, tag=None, inst_id=None, keep_original_names=False):
+def filter_metadata(meta_dict):
+    """Filter FUV metadata to remove warnings during loading.
+
+    Parameters
+    ----------
+    meta_dict : dict
+        Dictionary of metadata from file
+
+    Returns
+    -------
+    meta_dict : dict
+        Filtered FUV metadata
+
+    """
+
+    vars = ['ICON_L24_UTC_Time', 'ICON_L25_Inversion_Method']
+    for var in vars:
+        if var in meta_dict:
+            meta_dict[var]['FillVal'] = np.nan
+
+    vars = ['ICON_L25_Start_Times', 'ICON_L25_Stop_Times', 'ICON_L25_UTC_Time']
+    for var in vars:
+        if var in meta_dict:
+            meta_dict[var]['FillVal'] = np.nan
+            meta_dict[var]['ValidMin'] = np.nan
+            meta_dict[var]['ValidMax'] = np.nan
```

Comment:
Technically, shouldn't these be +/- np.inf?

Reply:
I have had issues when writing to netCDF when assigning `np.inf` before.